Source/WebCore/xml/XPathUtil.cpp

```diff
@@ -73,5 +74,10 @@ bool isValidContextNode(Node& node)
     return false;
 }
 
+bool isXMLSpace(UChar character)
+{
+    return character <= 0x20 && (character == 0x20 || character == 0x09 || character == 0x0D || character == 0x0A);
```

Comment:
Use `' '`, `'\t'` and such as we do elsewhere in the code?

(This also matches the definition of "HTTP space", might provide opportunity for further consolidation down the line.)

Reply:
Sure, I will fix it once EWS finishes and also will fix style error. If needed any help, I will ask. :-)

Reply:
Done.